modules/avoindata-drupal-newsfeed/templates/avoindata_newsfeed_block.html.twig

```diff
@@ -18,7 +18,7 @@
       {% for news in newsfeed %}
         <div class="avoindata-feeditem avoindata-feeditem-event">
           <a class="feeditem-date" href={{ path('entity.node.canonical', {'node': news.id}) }}>
-            {{ news.createdtime|format_date('avoindata_long_date') }}
+            {{ news.createdtime | format_date('avoindata_long_date') }}
```

Comment:
This does not need date('U')?

Reply:
Because created time is already in unix timestamp format.